nevergrad/benchmark/experiments.py

```diff
@@ -56,6 +59,7 @@ def mltuning(seed: tp.Optional[int] = None, overfitter: bool = False, seq: bool
     # First, a few functions with constraints.
     optims = ["Shiwa", "DE", "DiscreteOnePlusOne", "PortfolioDiscreteOnePlusOne", "CMA", "MetaRecentering",
               "DoubleFastGADiscreteOnePlusOne", "PSO", "BO", "MetaTuneRecentering"]
+    optims += ["Shiwa"] + ["ShiwaC" + str(i) for i in range(9)]
```

Comment:
from now on, please use optimizer groups and remove all the default_optims code, we can talk about it when you need guidance